react/__tests__/SliderLayoutGroup.test.tsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react'
+import {
+  render,
+  getByLabelText,
+  getByText,
+  fireEvent,
+} from '@vtex/test-tools/react'
+import { useResponsiveValue } from 'vtex.responsive-values'
+
+import SliderLayoutGroup from '../SliderLayoutGroup'
+import SliderLayout from '../SliderLayout'
+
+const mockedUseResponsiveValue = useResponsiveValue as jest.Mock<number>
+
+mockedUseResponsiveValue.mockImplementation(() => 1)
+
+test('should move all sliders in a group together', () => {
```

Comment:
Why not use `it` and wrap it up into a `describe`?

Reply:
Mostly because this is the only test for this component 😅

Reply:
Yes, but you have 2 blocks of `expect`, couldn't this be split into two tests?

Reply:
I guess they could be split, but they're testing pretty much the same thing, right? Checking if the sliders would actually slide. The difference here would just be to which side they're sliding.